internal/proxy/oauthproxy.go

```diff
@@ -589,6 +610,9 @@ func (p *OAuthProxy) OAuthCallback(rw http.ResponseWriter, req *http.Request) {
 	logger.WithRemoteAddress(remoteAddr).WithUser(session.Email).WithInGroups(session.Groups).Info(
 		fmt.Sprintf("oauth callback: user validated "))
 
+	//TODO: is there any verification we want to do on this?
```

Comment:
I think this is sufficient -- let's drop the TODO and a comment about why we add this new value to session